.vscode/tasks.json

```diff
@@ -71,10 +86,24 @@
       },
       "problemMatcher": []
     },
+    {
+      "label": "hassfest",
+      "type": "shell",
+      "command": "python3 -m script.hassfest && python3 -m script.hassfest -p metadata && python3 -m script.hassfest -p mypy_config",
```

Comment:
The `metadata` and `mypy_config` plugins already run on the first hassfest command; those are duplicates.

Reply:
Just taken from the pre-commit config, it explicitly runs both. I can easily remove the second two though.